exasol_transformers_extension/udfs/sequence_classification_single_text_udf.py

```diff
@@ -0,0 +1,44 @@
+import torch
+import transformers
+from exasol_transformers_extension.udfs import bucketfs_operations
+
+
+class SequenceClassificationSingleText:
+    def __init__(self,
+                 exa,
+                 cache_dir=None,
+                 base_model=transformers.AutoModelForSequenceClassification,
+                 tokenizer=transformers.AutoTokenizer):
+        self.exa = exa
+        self.cache_dir = cache_dir
+        self.base_model = base_model
+        self.tokenizer = tokenizer
+
+    def run(self, ctx):
+        bucketfs_conn = ctx.bucketfs_conn
```

Comment:
You need to iterate over the input and if possible we want to use batch prediction, but careful, the model could change. It is probably a good idea to add a order by on bucketfsconn and model parameters of the UDF.